sig-creation-procedure.md

```diff
@@ -0,0 +1,83 @@
+### SIG creation procedure
+
+#### 
+**Create the GitHub teams**
+
+Create the GitHub teams at [https://github.com/orgs/kubernetes/new-team](https://github.com/orgs/kubernetes/new-team), using the following name convention:
+
+* sig-foo-misc 
+* sig-foo-test-failures
+* sig-foo-bugs
+* sig-foo-feature-requests
+* sig-foo-proposals
+* sig-foo-pr-reviews
+* sig-foo-api-reviews
+
+I.e:
+
+* sig-onprem-misc
+* sig-onprem-test-failures
+* sig-onprem-bugs
+* sig-onprem-feature-requests
+* sig-onprem-proposals
+* sig-onprem-pr-reviews
+* sig-onprem-api-reviews
+
+#### 
+**Google Groups creation**
+
+Create Google Groups at [https://groups.google.com/forum/#!creategroup](https://groups.google.com/forum/#!creategroup), following the procedure: 
+
+* Create a group using the name convention below;
+* Group has to be created as an e-mail list with at least three owners (including sarahnovotny at google.com and ihor.dvoretskyi at gmail.com);
+* To add the owners, visit the Group Settings (drop-down menu on the right side), select Direct Add Members on the left side and add Sarah and Ihor via email address (with a suitable welcome message); in Members/All Members select Ihor and Sarah and assign to an "owner role";
+* Set "View topics", "Post", "Join the Group" permissions to be "Public"
+
+
+Name convention:
+
+* kubernetes-sig-foo-misc
```

Comment:
it looks like you have examples of the convention for each section but you do not explicitly definite the convention ever.  Something like "Names should follow the convention `kubernetes-sig-<YourSigName>`  where YourSigName has individual words divided by slashes."